subprojects/test-kit/src/integTest/groovy/org/gradle/testkit/runner/GradleRunnerFileSystemWatchingIntegrationTest.groovy

```diff
@@ -38,6 +37,7 @@ class GradleRunnerFileSystemWatchingIntegrationTest extends BaseGradleRunnerInte
         assumeTrue("File system watching is enabled by default", gradleVersion >= GradleVersion.version("7.0"))
     }
 
+    @NoDebug
```

Comment:
Why is this needed?

Reply:
This test only works when using `debug = false` on `GradleRunner`. I moved the `@NoDebug` from the class to the individual methods so I can also have a test for the behavior with `debug = true` in the same class.